.travis.yml

```diff
@@ -12,8 +12,10 @@ env:
 matrix:
  include:
   - python: 3.6
-    env: TOXENV=quality,docs
-    python: 3.6
+    env: TOXENV=quality
+  - python: 3.6
+    env: TOXENV=docs
+  - python: 3.6
```

Comment:
We have 2.7 and 3.5 above.

Reply:
What are you suggesting?

Reply:
I simplified this a bit; hopefully in line with what you were thinking.